OakCityV2/TableViewController.m

```diff
@@ -0,0 +1,216 @@
+//
+//  TableViewController.m
+//  OakCityV2
+//
+//  Created by Josh Green on 6/8/15.
+//  Copyright (c) 2015 Josh Green. All rights reserved.
+//
+
+#import "TableViewController.h"
+#import "AFNetworking/AFNetworking.h"
+#import "DetailViewController.h"
+#import "DetailCell.h"
+#import "Job.h"
+#import "AFNetworking/UIImageView+AFNetworking.h"
+#define LANGUAGE @"PHP"
+#define TOWN @"San+Francisco"
+
+@interface TableViewController ()
+
+@end
+
+@implementation TableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    //Fetch JSON
+    NSString *urlAsString = [NSString stringWithFormat:@"https://jobs.github.com/positions.json?description=%@&location=%@", LANGUAGE, TOWN];
+    NSURL *url = [NSURL URLWithString:urlAsString];
+    NSURLRequest *request = [NSURLRequest requestWithURL: url];
+    AFHTTPRequestOperation *operation = [[AFHTTPRequestOperation alloc] initWithRequest:request];
+    operation.responseSerializer = [AFJSONResponseSerializer serializer];
+    
+    //Parse JSON
+    [operation setCompletionBlockWithSuccess:^(AFHTTPRequestOperation *operation, id responseObject) {
+        self.jobs  = [NSMutableArray new];
+        NSArray *responseArray = (NSArray *)responseObject;
+        
+        for (NSDictionary *responseDict in responseArray) {
+        Job* job = [[Job alloc] initWithDictionary:responseDict];
+        [self.jobs addObject:job];
+        }
+        
+        if([self.jobs count] > 0) {
+            [self.tableView reloadData];
+        }
+
+        //Alert on failure to fetch JSON
+        else {
+            UIAlertView* alert = [[UIAlertView alloc]
+                                  initWithTitle: @"Failed to retrieve data" message: nil delegate: self
+                                  cancelButtonTitle: @"cancel" otherButtonTitles: @"Retry", nil];
+            [alert show];
+        }
+    }
+     
+     //Alert on fundemental errors
+                                     failure:^(AFHTTPRequestOperation *operation, NSError *error) {
+                                         UIAlertView *aV = [[UIAlertView alloc]
+                                                            initWithTitle:@"Error" message:[error localizedDescription] delegate: nil
+                                                            cancelButtonTitle:@"Ok" otherButtonTitles:nil];
+                                         [aV show];
+                                     }];
+    [operation start];
+    
+}
+
+//Alert View
+- (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)buttonIndex {
+    NSString *button = [alertView buttonTitleAtIndex:buttonIndex];
+    
+    if([button isEqualToString:@"Retry"]) {
+        //Fetch JSON
```

Comment:
Would you like to me work on implementing SDCAlertView or would you like to me to switch on the buttonIndex since there are a lot of other things to do currently?